src/Builder/RouteInfo.php

```diff
@@ -70,6 +70,17 @@ public function isMapped(): bool
         return $this->route->getOption('mapped') ?? $this->isVisible();
     }
 
+    /**
+     * Does the route relates to a content that might have dots in their slug?
+     *
+     * Set the `content.dots` option to true if some of the contents contains dots in their slug,
+     * which might wrongly be considered as an extension for the generated page.
+     */
+    public function hasDotsSupport(): bool
+    {
+        return $this->route->getOption('content')['dots'] ?? false;
```

Comment:
What about gathering/namespacing the `mapped` and `visible` options as well?

Reply:
Good idea! I didn't think of that way of "namespacing" all content-related option in an array but it makes sens.